tests/ha/single_pod_restart/test_data_server_pod_restart.py

```diff
@@ -518,3 +524,450 @@ def test_io_data_server_pod_restart(self):
+        thread_rd.daemon = True  # Daemonize thread
+        thread_rd.start()
+        LOGGER.info("Step 8: Successfully started READs and verify on the written data when pods "
+                    "down in step-7 in background")
+        LOGGER.info("Waiting for %s seconds", HA_CFG["common_params"]["30sec_delay"])
+        time.sleep(HA_CFG["common_params"]["30sec_delay"])
+
+        LOGGER.info("Step 9: Restore data and server pod and check cluster status.")
+        event.set()
+        for pod_prefix in self.pod_dict:
+            self.restore_method = self.pod_dict.get(pod_prefix)[-1]
+            resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                           restore_method=self.restore_method,
+                                           restore_params={
+                                               "deployment_name": self.pod_dict.get(pod_prefix)[-2],
+                                               "deployment_backup": self.deployment_backup,
+                                               "num_replica": self.pod_dict.get(pod_prefix)[2],
+                                               "set_name": self.pod_dict.get(pod_prefix)[1]},
+                                           clstr_status=True)
+            LOGGER.debug("Response: %s", resp)
+            assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+            LOGGER.info("Successfully restored pod by %s way", self.restore_method)
+        LOGGER.info("Step 9: Successfully started data and server pod and cluster is online.")
+        self.restore_pod = False
+        event.clear()
+        thread_rd.join()
+
+        LOGGER.info("Step 10: Verify status for In-flight READs/Verify DI")
+        responses_rd = dict()
+        while len(responses_rd) != 2:
+            responses_rd = output_rd.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_rd["pass_res"])
+        fail_logs = list(x[1] for x in responses_rd["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]), "READs/VerifyDI logs which contain failures:"
+                                                f"{resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs)
+        assert_utils.assert_false(len(resp[1]), "READs/VerifyDI Logs which contain failures: "
+                                                f"{resp[1]}")
+        LOGGER.info("Step 10: Verified status for In-flight READs/Verify DI")
+
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Step 11: Create new IAM user and buckets, Perform WRITEs-READs-Verify "
+                        "with variable object sizes after pod restart")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.test_prefix = f'test-45536-restart-{int(perf_counter_ns())}'
+            self.s3_clean.update(users)
```

Comment:
You need to run IOs also.

Reply:
IOs are running, just Logger is there in else statement